nevergrad/functions/stsp/core.py

```diff
@@ -16,9 +16,10 @@
 
 
 class STSP(ExperimentFunction):
-    def __init__(self, dimension: int = 500) -> None:
+    def __init__(self, dimension: int = 500, complex_tsp: bool = False) -> None:
```

Comment:
maybe just `complex` ? it's how you name it below + the class already states that it's tsp